build/lib/stats.js

```diff
@@ -116,7 +116,7 @@ function submitAllStats(productJson, commit) {
                 }
             */
             appInsights.defaultClient.trackEvent({
-                name: 'monacoworkbench/packagemetrics',
+                name: `${productJson.quality !== 'stable' ? 'adsworkbench' : 'monacoworkbench'}/packagemetrics`,
```

Comment:
This looks unrelated. Did you intend to have this in this PR? Can we ignore / remove?

Reply:
Maybe a merge from master will remove this?

Reply:
Hmm this just did a merge. Happens everytime I build.

Reply:
Reverted